docker/coolify/build_images.sh

```
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Change directory to root Server directory for correct Docker Context
+cd "$(dirname "$0")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)